sdk/core/azure-core/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - `Azure::Core::Http::Url::AppendPath` now does not encode the input by default.
 
+### Other changes and Improvements
+
+- Prevent pipeline of length zero to be created.
```

Comment:
Add a bug fixes section and move this there.

Similar to https://github.com/Azure/azure-sdk-for-c/blob/master/CHANGELOG.md#bug-fixes